src/BoundingBoxCameraSensor.cc

```diff
@@ -85,6 +85,15 @@ class gz::sensors::BoundingBoxCameraSensorPrivate
   /// \brief Just a mutex for thread safety
   public: std::mutex mutex;
 
+  /// \brief True if camera is triggered by a topic
+  public: bool isTriggeredCamera = false;
+
+  /// \brief True if camera has been triggered by a topic
+  public: bool isTriggered = false;
+
+  /// \brief Topic for camera trigger
+  public: std::string triggerTopic = "";
```

Comment:
```suggestion
  public: bool isTriggeredCamera{false};

  /// \brief True if camera has been triggered by a topic
  public: bool isTriggered{false};

  /// \brief Topic for camera trigger
  public: std::string triggerTopic{""};
```

Reply:
Addressed in [d7bc027](https://github.com/gazebosim/gz-sensors/pull/322/commits/d7bc027df336296e9e27c52272e98b4154faf691)